providers/dns/easydns/easydns.go

```diff
@@ -56,7 +56,7 @@ func NewDefaultConfig() *Config {
 	}
 }
 
-// DNSProvider describes a provider for acme-proxy
+// DNSProvider is an implementation of the challenge.Provider interface.
```

Comment:
I always shorten this to "*X* implements the *Y* interface", because I find it more readable.

I don't know. I'll leave this to you.